pyproject.toml

```diff
@@ -48,16 +48,16 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = ">=3.7,<3.12"
-boto3 = ">=1.3.0,<2"
+boto3 = "^1.3.0"
```

Comment:
[Caret notation](https://python-poetry.org/docs/dependency-specification/#caret-requirements) literally means ">={this version},<{next major version}". I'm just updating the notation, but the meaning is the same.